enterprise/suppress-user/handler_test.go

```diff
@@ -13,8 +14,13 @@ import (
 	"github.com/rudderlabs/rudder-server/enterprise/suppress-user/model"
 )
 
+var (
+	ErrNoValueFound = errors.New("no value found")
+	ErrNilNil       = errors.New("both nil values returned")
+)
+
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Not required? We can remove this?
```suggestion
	rand.New(rand.NewSource(time.Now().UnixNano()))
```